### PR TITLE
kie-issues#1447: Create a new example package on kie-tools to showcase how to use the new DMN Editor Standalone

### LINKED ISSUE #1447: Create a new example package on kie-tools to showcase how to use the new DMN Editor Standalone
Create simple html/ts/js files that load and run the new DMN Editor Standalone in different scenarios, displaying its API.

### PR DESCRIPTION
**Pull request by thiagoelg**
Closes https://github.com/apache/incubator-kie-issues/issues/1447

~~This was built on top of https://github.com/apache/incubator-kie-tools/pull/2538; that's why the diff is big.
Once that PR is merged, I'll merge `main` in this branch, making it more readable.~~

### CODE REVIEW

**Comment by jomarko** [comment]
Looking forward to doing a review once the original PR is merged.

**Comment by tiagobento** [comment]
@thiagoelg Please add a nice README.md to the new package...

File: examples/dmn-editor-standalone-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- [With Included Models]("src/withIncludedModels/index.ts"): Displays how Decisions can be imported into other Decisions as "Included Models", using the `resources` parameter from the DMN Editor.
+
+### Build
+
+To build the examples, execute one of the following commands on the root folder of the project:
+
+```shell script
+KIE_TOOLS_BUILD__buildExamples=true pnpm -F @kie-tools-examples/dmn-editor-standalone-examples... build:dev
+KIE_TOOLS_BUILD__buildExamples=true pnpm -F @kie-tools-examples/dmn-editor-standalone-examples... build:prod
+```
```
Comment (LightGuard) [inline examples/dmn-editor-standalone-examples/README.md:35]: Do we have another README on how to setup the environment that we can reference here? Simply running pnpm without bootstrapping, for example, won't be enough. If we do have another README to reference, we should point users there before they run these commands.

Comment (thiagoelg) [reply examples/dmn-editor-standalone-examples/README.md:35]: The [root README](https://github.com/apache/incubator-kie-tools/blob/main/README.md) describes the bootstrap steps, as well as how to set up the necessary tools to build on kie-tools.
Idk if we need to mention it here though, as that README should be the starting ground for everyone that's using the kie-tools repo.

Maybe that's something we can discuss in this issue: https://github.com/apache/incubator-kie-issues/issues/866

**Review by tiagobento — APPROVED** [review]

**Review by ljmotta — APPROVED** [review]

**Pull request merged**